lib/counter_culture/reconciler.rb

```diff
@@ -61,6 +61,7 @@ class Reconciliation
 
       delegate :model, :relation, :full_primary_key, :relation_reflect, :polymorphic?, :to => :counter
       delegate *CounterCulture::Counter::CONFIG_OPTIONS, :to => :counter
+      delegate :primary_key, :to => :relation_class
```

Comment:
I got pretty confused by this `delegate`, particularly because there are also some local variables called `primary_key` that conflict with this. Can we remove that and just stick with the `relation_class.primary_key` calls? It's not that much more typing. 😄

Reply:
Happy to remove. I think at the time I didn't think there was any shadowing? Idk, but was obviously wrong there. Should be good now!